docs/DeveloperGuide.md

```diff
@@ -425,15 +432,17 @@ The following activity diagram summarizes what happens when a user executes the
 
 **Target user profile**:
 
-* Need a place to their store travel plans in an organised manner
-* prefer desktop apps over other types
-* CLI proficient
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Has a need to their store travel plans in an organised manner
+* Has a need to store activities that they may visit as a wishlist
+* Prefers desktop apps over other types
+* Can type fast
+* Prefers typing to mouse interactions
+* Is reasonably comfortable using CLI apps
```

Comment:
"with using"?

Reply:
No need the with, intellij correct me also...
![image](https://user-images.githubusercontent.com/65444547/98458792-4b49b280-21cf-11eb-87bf-7581352fc7c3.png)